### PR TITLE
LPS-156761 Fix Poshi Automation - master

### PR DESCRIPTION
**Pull request by ozysouza**
https://issues.liferay.com/browse/LPS-156761

@david-gutierrez-mesa 
@Tim-Cao

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

File: portal-web/test/functional/com/liferay/portalweb/macros/Navigator.macro [hunk 1]
```diff
@@ -99,6 +99,16 @@ definition {
...
+	macro gotoNavListItem {
+		var key_tab = "${navTab}";
+
+		AssertClick(
+			locator1 = "NavTab#TAB_NAVIGATION",
+			value1 = "${navTab}");
+
+		AssertElementPresent(locator1 = "NavTab#ACTIVE_TAB_NAVIGATION");
+	}
+
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/macros/Navigator.macro:111]: Use existing macro

**Comment by Tim-Cao** [comment]
See https://github.com/liferay-echo/liferay-portal/pull/8626

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**